tests/test_harvest_innov_stats_engine.py

```diff
@@ -32,49 +32,52 @@
 SOCA_3DVAR = '%Y%m%d%H%M%S/post/soca/3dvar/'
 NETCDF_HARVESTER_CONFIG__VALID = 'netcdf_harvester_config__valid.yaml'
 
-def test_run_innov_stats_harvester_for_date_range():
```

Comment:
just a thought... do we want to move all broken tests until a broken test file? that could clean up unit test files without losing track of tests to fix

Reply:
I vote we leave it commented out for now, then address it when we get around to the better testing epic. It'll look a little messy for now, but we won't have to worry about missing anything moving it back and forth.

Reply:
I agree with Kevin. moving it could be more confusing and most importantly, it would disrupt any development history. We also keep track of this in the jira board. I don't want it to be standard practice for us to just comment out tests, but this is an exceptional circumstance.